### DISCUSSION TITLE
upload_to_testflight ignore FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD

### DISCUSSION THREAD

**Discussion by uchar** [fastlane/fastlane#18138] [Q&A]
I'm using `upload_to_testflight(skip_waiting_for_build_processing:true, skip_submission:true)` on Github Actions , but it ask me for 6 digits code every time it tries to build and fail with this error :

`[12:45:16]: 2FA can only be performed in interactive mode`

I add `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD` but get the same error.

With the new changes in apple developer, I can't use an account with 2fa/disabled ( Actually I can't even add an account with enough access because of https://developer.apple.com/forums/thread/114384) 

I'm stuck at making the build works for a couple of days now. Can someone help me?

**Comment by Yke222** [comment]
I'm currently facing the same issue.
